rpcserver.go

```diff
@@ -1832,6 +1838,53 @@ func GetUnconfirmedBalance(w *wallet.Wallet, chainSvr *chain.Client, icmd btcjso
 	return (unconfirmed - confirmed).ToBTC(), nil
 }
 
+// ImportAddress handles an importaddress request by parsing
+// a encoded address and importing it as a watch-only address to the
+// imported address account.
+func ImportAddress(w *wallet.Wallet, chainSvr *chain.Client, icmd btcjson.Cmd) (interface{}, error) {
+	cmd := icmd.(*btcjson.ImportAddressCmd)
+
+	addr, err := btcutil.DecodeAddress(cmd.Address, activeNet.Params)
+	if err != nil {
+		return nil, btcjson.Error{
+			Code:    btcjson.ErrInvalidAddressOrKey.Code,
```

Comment:
Check the struct definitions at the top of this file.  This can be simplified by simply doing:

```Go
return nil, InvalidAddressOrKeyError{err}
```